fdbclient/MonitorLeader.actor.cpp

```diff
@@ -468,7 +550,10 @@ ACTOR Future<MonitorLeaderInfo> monitorLeaderOneGeneration( Reference<ClusterCon
 	std::vector<Future<Void>> actors;
 	// Ask all coordinators if the worker is considered as a leader (leader nominee) by the coordinator.
 	for(int i=0; i<coordinators.clientLeaderServers.size(); i++)
-		actors.push_back( monitorNominee( coordinators.clusterKey, coordinators.clientLeaderServers[i], &nomineeChange, &nominees[i] ) );
+		actors.push_back(monitorNominee(
```

Comment:
this function should take a hostname and resolve it internally, to prevent the failure of one coordinator from restarting all actors for other coordinators

Reply:
Done.